src/containers/share-modal/components/ShareDrawer.tsx

```diff
@@ -17,12 +21,27 @@ export const ShareDrawer = () => {
   const { toast } = useContext(ToastContext)
   const dispatch = useDispatch()
   const track = useSelector(getTrack)
+  const artist = useSelector((state: CommonState) =>
+    getUser(state, { id: track?.owner_id })
+  )
   const source = useSelector(getSource)
 
   return (
     <ActionDrawer
       title='Share Track'
       actions={[
+        {
+          text: 'Share to Twitter',
+          icon: <IconTwitterBird height={20} width={26} />,
+          className: styles.shareToTwitterAction,
+          onClick: () => {
+            if (track && artist) {
```

Comment:
might be worth an else that does a console.error. we started piping those to sentry recently, which is nice!